anomalib/models/stfpm/lightning_model.py

```diff
@@ -27,22 +28,19 @@ class Stfpm(AnomalyModule):
 
     Args:
         input_size (Tuple[int, int]): Size of the model input.
-        backbone (str): Backbone CNN network
-        layers (List[str]): Layers to extract features from the backbone CNN
+        student_teacher_model (FeatureExtractorParams): Teacher model parameters.
```

Comment:
Can you remind me the main motivation of naming this as `student_teacher_model`? Can't this be `teacher_model`?

Reply:
Looks like I forgot to update the description of this argument, but the idea is that since we use the same backbone for both student and teacher model, the argument should indicate the same.